inc/class-yoast-acf-analysis-registry.php

```diff
@@ -13,11 +13,11 @@ class Yoast_ACF_Analysis_Registry {
 	private $storage = array();
 
 	/**
-	 * @param string|int                       $id    Registry index.
-	 * @param Yoast_ACF_Analysis_Configuration $class Registry value.
+	 * @param string|int $id   Registry index.
+	 * @param mixed      $item Item to store in the registry.
 	 */
-	public function add( $id, Yoast_ACF_Analysis_Configuration $class ) {
-		$this->storage[ $id ] = $class;
+	public function add( $id, $item ) {
```

Comment:
While this makes sense just looking at this class you can now put *anything* in the registry because [`boot()`](https://github.com/Yoast/yoast-acf-analysis/blob/7e1de670bcd76410d48fae3ef081b6a325687fed/inc/class-yoast-acf-analysis.php#L46-L58) "trusts" the registry.
Admittedly this should probably stay like this but needs to be fixed in `boot()`with and additional check like

`$registry->get( 'config' ) instanceof Yoast_ACF_Analysis_Configuration`